### ISSUE TITLE
Formatting Toolbar Active State Disappears

### ISSUE THREAD

**Issue by steven-tey** [TypeCellOS/BlockNote#155]
Probably need to persist hover-state classes and show them in the active state as well:

![CleanShot 2023-03-24 at 19 05 43](https://user-images.githubusercontent.com/28986134/227664523-b1b335bb-9382-48eb-9a29-d7a409dd9055.gif)

**Comment by YousefED** [comment]
I think this is probably caused by the added Tailwind styles, there must be a conflict going in. Any thoughts what it could be? I think I'm able to reproduce, so will check as well

**Comment by steven-tey** [comment]
Good point, since it doesn't seem to be happening in https://blocknote-main.vercel.app/...

However, on closer look, even if I removed the Tailwind styles (`editorDOMAttributes` and `defaultStyles` override), the bug is still there <picture data-single-emoji=":thonkhard:" title=":thonkhard:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..wrvEoLVh8IyPof5q.jbwhZpRPw_Y0fHA8QWhyLPYIDSGCiMTBZfzvKt05dQ0IJl_3VmxmxZiXZEI013oFljMDNAeKcL594hX4wZsB0A965gyWUEa7k3Lp4-f4RqFrL2qwZbi0H_Q_.qGX-OPNvyLl2tEGcJretzA" alt=":thonkhard:" align="absmiddle"></picture>

**Comment by YousefED** [comment]
What if you remove the tailwind stylesheets? Maybe some globals are affected

**Comment by steven-tey** [comment]
Ahh yup, you're right! Some of the Tailwind globals are conflicting with the Mantine classes

**Comment by steven-tey** [comment]
Another odd thing is if I set `prose-h1:text-2xl`, the font size changes but the transition/resulting div doesn't change with it:

![CleanShot 2023-03-24 at 22 35 36](https://user-images.githubusercontent.com/28986134/227689954-414062de-4cc8-4ae5-82f2-9df8170eaa92.gif)

Should I open up a new issue for this?

**Comment by sasa95** [comment]
> Ahh yup, you're right! Some of the Tailwind globals are conflicting with the Mantine classes

+1

This style from `@tailwind base `overrides the Mantine class:

```css
button, [type='button'], [type='reset'], [type='submit'] {
    -webkit-appearance: button;
    background-color: transparent; /* this one */
    background-image: none;
}`
